Source/Charts/Renderers/XAxisRenderer.swift

```diff
@@ -12,9 +12,6 @@
 import Foundation
 import CoreGraphics
 
-#if !os(OSX)
```

Comment:
we can safely remove this for macOS?

Reply:
Foundation and CG are the only two frameworks used. UIKit must have been used a long time ago and never removed.